cli.py

```diff
@@ -185,11 +186,20 @@ def _add_export_output_path_args(parser) -> None:
         default=None,
         help="Output to the specified AOT Inductor .dso model file",
     )
+
+
+def _add_export_args(parser) -> None:
     parser.add_argument(
         "--dynamic-shapes",
         action="store_true",
         help="Call torch.export with dynamic shapes",
     )
+    parser.add_argument(
+        "--max-seq-length",
+        type=int,
+        default=None,
+        help="Set maximum length sequence when before calling torch.export",
```

Comment:
set the default to 300 and update the help string so that it's clear what the default is (300)

Reply:
That was my initial implementation. Then there was an issue when running eval. 

When running eval, we use `--dynamic-shapes` which uses a larger max_seq_length, i.e. model.config.max_seq_length. But in theory, we should not stop user from calling eval with both options, something like `--dynamic-shapes --max-seq-length 1000`.  When that happens, if args.max_seq_length has a default value, we will have no way to distinguish if args.max_seq_length is from a default value or from an intentional user overwriting.